coding-guides/making-your-own-starboard.md

```diff
@@ -90,7 +90,7 @@ if (stars) {
   // We fetch the ID of the message already on the starboard.
```

Comment:
So GitHub doesn't let me comment on code that's not by your changes but on line 64 here I noticed that we casually mention regex, without explaining it. This seems weird considering as far as I'm aware we don't really talk about it elsewhere. Maybe something simple like

> Regex is a special pattern that is used for finding and parsing text. For more info about regex see: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions